framework/README.md

```diff
@@ -4,6 +4,7 @@
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/3171affc728048da8df4fe36b6d4771e)](https://app.codacy.com/gh/Idrinth/api-bench/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade)
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=Idrinth_api-bench&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=Idrinth_api-bench)
 
+
```

Comment:
why an added empty line? Wouldn't the text from the root readme fit in here well?

Reply:
hey @Idrinth can you please mention specifically(if possible file route) where to add " what is "

I'm confused with the documentation thank you.

Reply:
hi @akhil0203 the documentation hompage is in:
```
documentation-website/src/pages/home/index.tsx
```
The text goes into
```
documentation-website/language/en.yml
```

Reply:
thanks a lot @Idrinth